src/hotspot/share/classfile/stringTable.cpp

```diff
@@ -650,6 +654,32 @@ size_t StringTable::verify_and_compare_entries() {
   return vcs._errors;
 }
 
+static void print_string(Thread* thr, outputStream* st, oop s) {
```

Comment:
Nit: if `thr` is the current thread (has to be for the `ResourceMark`) then please call it `current`.

Reply:
Fixed.